arches_vue_utils/src/arches_vue_utils/utils.spec.ts

```diff
@@ -17,6 +17,11 @@ function asLabel(valuetype_id: string, language_id: string): Label {
 }
 
 const systemLanguageCode = "en-ZA"; // arbitrary
+const emptyLabel = {
+    value: "",
```

Comment:
Elsewhere in Arches, we have `None` as a placeholder for no data (e.g. resource report page). What do you think of returning that instead of empty string to maintain the pattern and signal the absence of data?